client/signup/steps/design-picker/index.jsx

```diff
@@ -86,9 +86,10 @@ class DesignPickerStep extends Component {
 		const allThemes = this.props.themes
 			.filter( ( { id } ) => ! EXCLUDED_THEMES.includes( id ) )
 			.map( ( { id, name, taxonomies } ) => ( {
-				categories: taxonomies?.theme_subject ?? [
-					{ name: this.props.translate( 'No Category' ), slug: 'CLIENT_ONLY-no-category' },
-				],
+				categories: taxonomies?.theme_subject ?? [],
+				// Blank Canvas uses the theme_picks taxonomy with a "featured" term in order to
```

Comment:
Yes at the moment it is. This is the logic the gallery at `/themes/:siteSlug` uses to show Blank Canvas at the top, so I thought it'd be good to replicate that logic rather than special case the `blank-canvas` slug.